website/public/js/controllers/groupsCtrl.js

```diff
@@ -16,6 +16,15 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
     }
 
     $scope.isMemberOfGroup = function(userid, group){
+
+      // If the group is a guild, just check for an intersection with the
+      // current user's guilds, rather than checking the members of the group.
+      if(group.type === 'guild') {
+        if (_.detect(Groups.myGuilds(), function(g) { return g._id === group._id })) {
+          return true;
```

Comment:
Would it be worth returning false if the group is a guild and it's not in the myGuilds call? I'm thinking:
```
if(group.type === 'guild') {
  return _.detect(Groups.myGuilds(), function(g) { return g._id === group._id })
}
```